src/main/java/elements/CheckBoxImpl.java

```diff
@@ -0,0 +1,45 @@
+package elements;
+
+
+import org.openqa.selenium.ElementNotInteractableException;
+import org.openqa.selenium.ElementNotVisibleException;
+import org.openqa.selenium.WebElement;
+import util.ScrollUtils;
+import util.WaitUtils;
+
+
+/***
+ * Wrapper class with Checkbox functionalities
+ */
+public class CheckBoxImpl extends ElementImpl implements Checkbox {
+
+    public CheckBoxImpl(WebElement element) {
+        super(element);
+    }
+
+    @Override
+    public void clickCheckbox() {
```

Comment:
it's better to implement method clickCheckbox(boolean isChecked), where isChecked - state of checkbox which should be set

Reply:
^^

Reply:
done